src/actions/actionCreators.js

```diff
@@ -8,3 +8,44 @@ import * as types from './actionTypes';
   })
 
 */
+export const login = (username, password) => (dispatch) => {
```

Comment:
My understanding is that actionCreators.js should have functions that accept a payload as argument and returns an action object with properties 'type' and 'payload'. Is this a Redux Thunk requirement to make async requests?

Reply:
yees